templates/frontOffice/default/product.html

```diff
@@ -153,12 +153,15 @@ <h1 class="name"><span itemprop="name">{$TITLE}</span><span id="pse-name" class=
                             <meta itemprop="category" content="{$TITLE}">
                         {/loop}
                         {* List of condition : NewCondition, DamagedCondition, UsedCondition, RefurbishedCondition *}
-                        <meta itemprop="itemCondition" itemscope itemtype="http://schema.org/NewCondition">
+                        {if $IS_NEW}
```

Comment:
I don't think it's the same new. `newCondition` indicates that this product has not been used. The `IS_NEW` of Thelia indicates that this product is new in the catalog.